OpenRA.Mods.Common/Widgets/SupportPowersWidget.cs

```diff
@@ -244,7 +289,6 @@ public override void Draw()
 
 		public override void Tick()
 		{
-			// TODO: Only do this when the powers have changed
 			RefreshIcons();
```

Comment:
IMO it would be better to check `powersModified` here, and then have `RefreshIcons` always refresh icons when it is called.